src/Plugin.php

```diff
@@ -292,4 +295,13 @@ public static function renderTemplate(array $template_subpathnames, array $varia
     }
   }
 
+  /**
+   * Safari 5 and some mobile browsers do not support umlauts in image URLs yet.
+   *
+   * @implements sanitize_file_name
+   */
+  public static function sanitize_file_name($filename) {
+    return preg_replace('/[^A-Za-z0-9.,_-]+/', '', remove_accents($filename));
```

Comment:
Do you really need the regex? Isn't `remove_accents()` enough? https://github.com/WordPress/WordPress/blob/7004afe4f4bac1fd17a142051832bdf6be8e6fcf/wp-includes/formatting.php#L1603

Reply:
I went to [https://gitlab.com/netzstrategen/nland/-/commits/master/wp-content/plugins/custom/src/Plugin.php](https://gitlab.com/netzstrategen/nland/-/commits/master/wp-content/plugins/custom/src/Plugin.php), and saw Sunny's comments, he first used only remove_accents, and then rewrote the function like that and tagged as fixing for covering non-ASCII characters. I can take it out if you think I should, let me know...

Reply:
Ah, OK. Leave as it is then. 👍